pkg/controllers/provisioning/v1alpha1/reallocation/controller.go

```diff
@@ -70,14 +70,14 @@ func (c *Controller) Reconcile(object controllers.Object) error {
 		return fmt.Errorf("listing underutilized nodes, %w", err)
 	}
 
-	// TODO: Further filter underutilized nodes that haven't been cordoned/TTLed to not spam logs
-	if len(underutilized) != 0 {
-		zap.S().Debugf("Found %d underutilized nodes", len(underutilized))
+	// Filter underutilized nodes to those that haven't had TTL set
+	nonTTLed := c.filter.GetTTLableNodes(underutilized)
+	if len(nonTTLed) != 0 {
+		zap.S().Debugf("Found %d underutilized non-ttled nodes", len(nonTTLed))
```

Comment:
If we're emitting a TTL statement for each node, do we need this one? Cleaner without IMO

Reply:
Hmm. You make a good point. This statement makes it clearer exactly how many are being actioned on. It doesn't add any additional information, but organizes it a little better. I like keeping it, unless you think otherwise.